ksrates/fc_plotting.py

```diff
@@ -67,7 +67,7 @@ def generate_mixed_plot_figure(species, x_max_lim, y_max_lim, corrected_or_not,
     if plot_correction_arrows and correction_table_available:
         fig, ax = plt.subplots(1, 1, figsize=(14.0, 7.6))
```

Comment:
```suggestion
    if correction_table_available:
        if plot_correction_arrows:
            fig, ax = plt.subplots(1, 1, figsize=(14.0, 7.6))
        else:
            fig, ax = plt.subplots(1, 1, figsize=(14.0, 7.0))
```

Reply:
I can't make suggestions to all lines of code but this also needs to be added to the end of this function:
```
    if not correction_table_available:
        plt.tight_layout()

    return fig, ax
```